src/contracts/core/DelegationManager.sol

```diff
@@ -373,12 +392,7 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
             address operator = delegatedTo[staker];
 
             // subtract strategy shares from delegate's shares
-            _decreaseOperatorShares({
-                operator: operator,
-                staker: staker,
-                strategy: strategy,
-                shares: shares
-            });
+            _decreaseOperatorShares({operator: operator, staker: staker, strategy: strategy, shares: shares});
```

Comment:
Oof. This one is pretty bad.

Reply:
I see it's trying to make this a single line because it fits in the line length, but when we use explicit parameter-named syntax like this, it's almost universally clearer to have each param on a single line

Reply:
Yeah not a huge fan either...

I'll see if there's a way to avoid it via `foundry.toml`

#### Possible solutions
1) Use Foundry's comment flags like this:
```
// forgefmt: disable-next-item 
```
2) Maybe reduce max line length a bit?